docs/templates/docker-in-docker.md

```diff
@@ -219,32 +240,31 @@ resource "coder_agent" "main" {
 }
 ```
 
-### Use systemd in Kubernetes-based templates:
-
-After [installing Sysbox on Kubernetes](https://github.com/nestybox/sysbox/blob/master/docs/user-guide/install-k8s.md), modify your template to use the sysbox-runc RuntimeClass.
+### Use systemd in Kubernetes-based templates
 
-> Currently, the official [Kubernetes Terraform Provider](https://registry.terraform.io/providers/hashicorp/kubernetes/latest) does not support specifying a custom RuntimeClass. [mingfang/k8s](https://registry.terraform.io/providers/mingfang/k8s), a third-party provider, can be used instead.
+After [installing Sysbox on Kubernetes](https://github.com/nestybox/sysbox/blob/master/docs/user-guide/install-k8s.md),
+modify your template to use the sysbox-runc RuntimeClass. This requires the Kubernetes Terraform provider version 2.16.0 or greater.
 
 ```hcl
 terraform {
   required_providers {
     coder = {
       source  = "coder/coder"
     }
-    k8s = {
-      source = "mingfang/k8s"
+    kubernetes = {
+      source = "hashicorp/kubernetes"
+      version = "2.16.0"
     }
   }
 }
 
-
 resource "coder_agent" "main" {
   os   = "linux"
   arch = "amd64"
   dir  = "/home/coder"
 }
 
-resource "k8s_core_v1_pod" "dev" {
+resource "kubernetes_pod" "dev" {
```

Comment:
The schema of the official provider is different, so we'll need to make some changes. This wouldn't work

Reply:
@bpmct am i missing something? the pod example in the official provider uses:

```hcl
resource "kubernetes_pod" "dev" {
  ...
}
```

see here: https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/pod#example-usage

Reply:
the actual properties are different inside the resource block